README.md

```diff
@@ -0,0 +1,27 @@
+# e2e-harness
+
+[![CircleCI](https://circleci.com/gh/giantswarm/e2e-harness.svg?style=svg)](https://circleci.com/gh/giantswarm/e2e-harness)
```

Comment:
I know mr timothy prefers `style=shield`, if you wouldn't mind updating

Reply:
sure, done